### DISCUSSION TITLE
Can't access Web Crypto API from within remix's dev server

### DISCUSSION THREAD

**Discussion by tavoyne** [remix-run/remix#6913] [Q&A]
Any idea why I can't access the Web Crypto API from within remix's dev server? I'm on Node `18.17.0`, so `globalThis.crypto` shouldn't be `undefined`. Tried with remix `1.16.1` and `1.19.0`.

**Comment by kiliman** [comment]
As discussed in #6915, you must be on Node.js v19+ to use `globalThis.crypto`.

**Comment by ABDULRAFEH123** [comment]
i also face the same issue  my  issue was when i run my project  i got error as like crypto is undefined .or somthhing like that..  ok.. so if you wnats to solve that issue then update your node version and use v19.0.0 ok.. this wil solve your problem .. its 100% work for me ok.. good luck
